tawk_to.module

```diff
@@ -244,33 +227,35 @@ function tawk_to_set_widget()
 {
 
     if (!isset($_POST['pageId']) || !isset($_POST['widgetId'])) {
-        return drupal_json_output(array('success' => false));
+        return backdrop_json_output(array('success' => false));
     }
 
     if (preg_match('/^[0-9A-Fa-f]{24}$/', $_POST['pageId']) !== 1 || preg_match('/^[a-z0-9]{1,50}$/i', $_POST['widgetId']) !== 1) {
-        return drupal_json_output(array('success' => false));
+        return backdrop_json_output(array('success' => false));
     }
 
-    variable_set(TAWKTO_WIDGET_PAGE_ID, $_POST['pageId']);
-    variable_set(TAWKTO_WIDGET_WIDGET_ID, $_POST['widgetId']);
+    config_set('tawk_to.settings', 'page_id', $_POST['pageId']);
+    config_set('tawk_to.settings', 'widget_id', $_POST['widgetId']);
 
     global $user;
     if ($user->uid) {
-        variable_set(TAWKTO_WIDGET_USER_ID, $user->uid);
+        config_set('tawk_to.settings', 'widget_user_id', $user->uid);
     }
 
-    drupal_json_output(array('success' => true));
+    backdrop_json_output(array('success' => true));
 }
 
 /**
  * Ajax endpoint which is used to remove currently selected widget details.
  */
 function tawk_to_remove_widget($a = 0, $b = 0)
 {
-    variable_del(TAWKTO_WIDGET_PAGE_ID);
-    variable_del(TAWKTO_WIDGET_WIDGET_ID);
+	$config = config('tawk_to.settings');
+	$config->clear('page_id');
+	$config->clear('widget_id');
+	$config->save;
```

Comment:
Just some nitpicks, we're using 4 spaces for indentation instead of tabs 😁

Reply:
Good point.  I've applied those and committed the other changes